recipes/cruft/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "cruft" %}
+{% set version = "2.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7c0f7682765e76fcf31adf877ea6f74372a0ab9554d8f8d6766e8e0413730e52
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-cache-dir -vvv"
+  entry_points:
+    - cruft = cruft._cli:app
+
+requirements:
+  host:
+    - python
+    - pip
+    - poetry >=0.12
+  run:
+    - python >=3.6
+    - cookiecutter >=1.6,<2.0
+    - gitpython >=3.0,<4.0
+    - typer >=0.3.1,<0.4.0
+    - click >=7.1.2,<8.0.0
+
+test:
+  requires:
+    - git
+  imports:
+    - cruft
+    - cruft._commands
+  commands:
+    - cruft --help
+
+about:
+  home: https://github.com/timothycrosley/cruft
```

Comment:
The project has moved to https://github.com/cruft/cruft

Reply:
I can open another PR and fix that very quickly.